app/models/claim.rb

```diff
@@ -262,6 +272,10 @@ def school
     eligibility&.current_school
   end
 
+  def amendable?
+    submitted? && !decision&.rejected? && !payment && !pii_removed?
```

Comment:
I think that `!payment` is checking that it hasn't been _payrolled_, not that it hasn't been _paid_. I'm pretty sure this is actually correct behaviour (ie to make an amendment someone must first remove the claim from a payroll run), but just so we're all clear.

Reply:
Yep, thanks for clarifying - realised that's what you meant. The commit message for this change makes it clear.